Model/Config.php

```diff
@@ -24,6 +24,7 @@ class Config
     private const PATH_INTEGRATION_API_URL = 'checkout/bold_checkout_payment_booster_advanced/api_url';
     private const PATH_LOG_IS_ENABLED = 'checkout/bold_checkout_payment_booster_advanced/log_enabled';
     private const PATH_SHARED_SECRET = 'checkout/bold_checkout_payment_booster/shared_secret';
+    private const PATH_PAYMENT_BOOSTER_FLOW_ID = 'checkout/bold_checkout_payment_booster/default_flow_id';
```

Comment:
[Clarification] I went back and forth between naming this `payment_booster_flow_id` and then default. I settled on default since this is the PB extension, but let me know others thoughts here.